gallery/src/data-visualization/ReactVisPage.tsx

```diff
@@ -0,0 +1,150 @@
+import { NxTable, NxTableBody, NxTableCell, NxTableHead, NxTableRow,
+  NxWarningAlert } from '@sonatype/react-shared-components';
+
+const ReactVisPage = () =>
+  <>
+    <GalleryDescriptionTile >
+      <p className="nx-p">
+        React-vis is a React-based data visualization library. It is highly customizable, easy-to-use, and very
+        React-friendly. The library is composed of React components that are used to render common data visualization
+        charts like line/area/bar charts, scatterplots, pie and donut charts etc.
+      </p>
+      <h3 className="nx-h2">React-vis Basics</h3>
+      <h3 className="nx-h3">XYPlot</h3>
+      <p className="nx-p">
+        Every series chart that React-vis generates (excludes RadialChart, Sankey, Sunburst and Treemap)
+        is inside a parent component called <code className="nx-code">XYPlot</code>
+        that requires two props: <code className="nx-code">height</code> and <code className="nx-code">width</code>.
+      </p>
+      <p className="nx-p">
+        <code className="nx-code">{'<XYPlot height={300} width = {300} />'}</code>
+      </p>
+      <h3 className="nx-h3">Ordering</h3>
+      <p className="nx-p">
+        Similar to traditional React, order matters as components are drawn in order. For example, placing
+        <code className="nx-code">{'<VerticalGridLines />'}</code> before
+        <code className="nx-code">{'<LineSeries data={data} />'}</code> will draw the chart on top of the gridlines.
+      </p>
+      <h3 className="nx-h3">Colors</h3>
+      <p className="nx-p">
+        Colors are assigned to charts by passing in the <code className="nx-code">color</code> or the
+        <code className="nx-code">stroke</code> prop depending on the type of chart being used.
+      </p>
+      <p className="nx-p">
+        <code className="nx-code">{'<LineSeries />'}</code> charts use the <code className="nx-code">stroke</code>
+        {' '} prop to determine colors of the lines.
+      </p>
+      <p className="nx-p">
+        <code className="nx-code">{'<BarSeries />'}</code> charts use the <code className="nx-code">color</code>
+        {' '} prop to determine colors of the bars.
+      </p>
+      <p className="nx-p">
+        <code className="nx-code">{'<RadialChart />'}</code> uses the <code className="nx-code">color</code>
+        {' '} property on the <code className="nx-code">data</code> object to determine colors of the bars.
+      </p>
+      <p className="nx-p">
+        RSC's <code className="nx-code">{'selectableColors'}</code> array provides a seamlessly way of
+        assigning colors based on RSC color guidelines. Usage is quite simple. Simply import the
+        <code className="nx-code">{'selectableColors'}</code> array and pass it on the
+        <code className="nx-code">stroke</code> or the <code className="nx-code">color</code> prop. An example
+        of using the color purple (<code className="nx-code">selectableColors[0]</code>) is shown below:
```

Comment:
This doesn't work.  `selectableColors[0]` does in fact currently evaluate to "purple", but passing "purple" as the `stroke` causes the line color to be the CSS value "purple" (i.e. the browser's built-in purple #800080) and not the RSC purple selectable color (either `hsl(280, 100%, 80%)` or `hsl(280, 100%, 95%)` depending on whether you meant the light or dark variant).

Reply:
> This doesn't work. `selectableColors[0]` does in fact currently evaluate to "purple", but passing "purple" as the `stroke` causes the line color to be the CSS value "purple" (i.e. the browser's built-in purple `#800080`) and not the RSC purple selectable color (either `hsl(280, 100%, 80%)` or `hsl(280, 100%, 95%)` depending on whether you meant the light or dark variant).

Changed to use `selectableColorClasses` instead.
65093a218837d323181c3b2788135f6ca536421a